client/src/components/Cart/Total.js

```diff
@@ -51,9 +51,8 @@ const Total = (props) => {
   const items = props.items;
   const title = `(Rs.${props.price})`;
   const classes = CartStyles();
-
+  const state = useAuthState();
```

Comment:
We are not using this state anywhere. If it's not needed, we should remove this.

Reply:
We are using it

Reply:
https://github.com/DSC-VJTI/CA-Website/blob/main/client/src/components/Cart/Total.js
Line 70

I got an error for the same that state is not defined, when I was trying to run the client side, using state solved it as well

Reply:
Okay, never noticed that. Works